### ISSUE TITLE
Team sync - June 27, 2024

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/czi-scientific-python-mgmt#104]

**Comment by trallard** [comment]
@gabalafou to check https://github.com/pydata/pydata-sphinx-theme/pull/1900

**Comment by gabalafou** [comment]
Updates for Post PST A11y Audit:

- Gabriel
  - Thanks to Matthias's review, was able to close out the issue with [navbar icons' small hit area](https://github.com/orgs/Quansight-Labs/projects/11/views/7?pane=issue&itemId=64994348)
- Matthias
  - Reviewing PRs and orienting himself to the project
- Smera 
  - Has done research for [Sphinx copy button](https://github.com/orgs/Quansight-Labs/projects/11/views/7?pane=issue&itemId=64955760)
  - Has also been working on [chevron hover effect](https://github.com/orgs/Quansight-Labs/projects/11/views/7?pane=issue&itemId=64956834)
- Isabela
  - Has been out. Knows which tasks are assigned. But no progress yet.

**Comment by gabalafou** [comment]
Tania asked me for update on [Axe failures](https://github.com/pydata/pydata-sphinx-theme/issues/1428#ref-commit-1c338cb).

They are almost all addressed. As soon as I can debug and [fix a race condition](https://github.com/pydata/pydata-sphinx-theme/pull/1905#issuecomment-2195185004), all of the automated tests will pass or be marked as expected failures, which should pave the way to adding these accessibility checks to the CI.

**Comment by gabalafou** [comment]
Copy button discussion

Various ideas discussed, such as, does every code block really need a copy button? In that case, why not every paragraph?

Decision made:

- Smera will continue working on the issue until she has a solid design recommendation. This is to ensure that while sites continue to the use the copy button, we put our best foot forward with it in terms of accessibility.
- Matthias will continue working on an alternative solution to the copy button.
